src/framework/parsers/gsplat-resource.js

```diff
@@ -66,6 +66,12 @@ class GSplatResource {
             instance: splatInstance
         });
 
+        // the ply scene data no longer gets automatically rotated on load, so do
+        // it here instead.
+        if (!this.splatData.isCompressed) {
+            entity.setLocalEulerAngles(0, 0, 180);
```

Comment:
not sure I'm keen on this, entity transform is owned by the user, we should not modify it. Better just have the entity upside down and the user needs to rotate it the way it needs to

Reply:
What's the danger in returning a correctly oriented entity?

It saves every user from having to worry about reorienting only uncompressed PLY (compressed PLY doesn't need this). And as you say, user owns the transform, they can do with it what they want.

It's either this or we create a child node with this transform, which I think is probably worse.

Reply:
They might have script that rotates it by specifying angles, and by default it would not work the same as for other entities, but would flip it upside down.

child entity is worse, agreed.
I'd say we do nothing here, let them handle it.
but ok, maybe pre-rotate it, and give trouble to smaller number of users, I guess I can live with it, even though I wont like it. I'd expect every entity the engine creates has identity transform.